.changeset/curvy-cats-roll.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/console": patch
```

Comment:
Need to patch theme module as well.

Reply:
Fixed with 392095343a92ec1af6b83208deeda3e3562093f1